scripts/rmc-regression.sh

```diff
@@ -25,7 +25,7 @@ check-cbmc-viewer-version.py --major 2 --minor 5
 ./scripts/setup/build_rmc_lib.sh
 
 # Standalone rmc tests, expected tests, and cargo tests
-./x.py test -i --stage 1 rmc firecracker prusti smack expected cargo-rmc
+./x.py test -i --stage 1 rmc firecracker prusti smack expected cargo-rmc rmc-docs
```

Comment:
I thought testing `rmc-docs` examples was going to be done by the `build-docs.sh` script (see the `TODO` comment there).

I'm okay with having it here but then we should remove the comment. Does the regression workflow need to change too? Please check.

Reply:
Ah good point! I'll remove the comment and add another bit of documentation about how to run the tests for the docs.

The CI workflow runs the `rmc-regression.sh` script.

Reply:
Right, but I recall having a discussion with you about the condition to execute `build-docs.sh`. Anyway, looks like it executes on Ubuntu 20.04 only - that is fine if we run them from `rmc-regression.sh`.